docs/guides/integration-prebid-client-side.md

```diff
@@ -144,6 +144,9 @@ pbjs.setConfig({
 }); 
 ```
 
+It is possible that the user's DII has opted out of UID2 previously. In this case, the UID2 module will respect the optout preference and no UID2 token will be generated and collected by Prebid.js.
```

Comment:
i don't think we need to mention much here apart from saying prebid.js UID2 module will handle optout, unlike javascript sdk as pub needs to retrieve the ad token manually in latter